uint16[] id_list

string[] drive_mode
string[] operation_mode
bool[]   torque_enable
